sdk/servicebus/azure-servicebus/api_samples.py

```diff
@@ -0,0 +1,317 @@
+
+    # 1.2 send batch
+    batch_message = topic_sender.create_batch(max_size_in_bytes=256 * 1204)
+    while True:
+        try:
+            batch_message.add(Message("Test message"))
+        except ValueError:
+            break
+
+    topic_sender.send(batch_message)
+
+    # 1.3 schedule
+    schedule_message = Message("scheduled_message")
+
+    enqueue_time_utc = datetime.utcnow() + timedelta(minutes=10)
+    sequence_number = topic_sender.schedule(schedule_message, enqueue_time_utc)
+
+    running = True
+
+    batch_schedule_message = topic_sender.create_batch(max_size_in_bytes=256 * 1204)
+
+    while running:
+        try:
+            batch_message.add(Message("Test message"))
+        except ValueError:
+            batch_sequence_numbers = topic_sender.schedule(batch_schedule_message, enqueue_time_utc)
+            batch_schedule_message = topic_sender.create_batch(max_size_in_bytes=256 * 1204)
+
+
+    # 1.4 cancel schedule
+    topic_sender.cancel_scheduled_messages(sequence_number)
+    topic_sender.cancel_scheduled_messages(batch_sequence_numbers)
+
+# 2 Receive
+sb_client = ServiceBusClient.from_connection_string(conn_str="conn_str")
+queue_receiver = sb_client.get_queue_receiver(queue_name="queue_name")
+
+# queue_receiver = ServiceBusReceiver(
+#     fully_qualified_namespace="fully_qualified_namespace",
+#     queue_name="queue_name",
+#     credential=ServiceBusSharedKeyCredential("policy", "key")
+# )
+
+'''
+subscription_receiver = ServiceBusReceiver(
+    fully_qualified_namespace="fully_qualified_namespace",
+    topic_name="topic_name",
+    subscription_name="subscription_name"
+    credential=ServiceBusSharedKeyCredential("policy", "key")
+)
+'''
+
+# 2.1 peek
+with queue_receiver:
+    msgs = queue_receiver.peek(count=10)
+    for msg in msgs:
+        print(msg)
+
+# 2.2 iterator receive
+with queue_receiver:
+    for msg in queue_receiver:
+        print(message)
+        msg.complete()
+        # msg.renew_lock()
+        # msg.abondon()
+        # msg.defer()
+        # msg.dead_letter()
+
+# 2.3 pull mode receive
+with queue_receiver:
+    msgs = queue_receiver.receive(max_batch_size=10)
+    for msg in msgs:
+        msg.complete()
+
+# 2.4 receive deferred letter
+with queue_receiver:
+    defered_sequence_numbers = [1,2,3,4,5,6]
+    deferred_messages = queue_receiver.receive_deferred_messages(defered_sequence_numbers)
+    for i in deferred_messages:
+        deferred_messages.abandon()
+
+
+### Session operation
+
+
+# 1 Send
+sb_client = ServiceBusClient.from_connection_string(conn_str="conn_str")
+topic_sender = sb_client.get_topic_sender(topic_name="topic_name")
+
+# topic_sender = ServiceBusSender(
+#     fully_qualified_namespace="fully_qualified_namepsace",
+#     topic_name="topic_name",
+#     credential=ServiceBusSharedKeyCredential("policy", "key")
+# )
+
+with topic_sender:
+    # 1.1 send single
+    message = Message("Test message")
+    # message.session_id = "test_session"
```

Comment:
batch contain multiple sessions?

remove `session_id` on the outgoing Message object

have more discussion with Service Team